deriv.com.conf

```diff
@@ -0,0 +1,55 @@
+map $http_cf_ipcountry $redirected_url {
+    BE /eu;
+    BG /eu;
+    CZ /eu;
+    DK /eu;
+    DE /eu;
+    EE /eu;
+    IE /eu;
+    EL /eu;
+    ES /eu;
+    FR /eu;
+    HR /eu;
+    IT /eu;
+    CY /eu;
+    LV /eu;
+    LT /eu;
+    LU /eu;
+    HU /eu;
+    MT /eu;
+    NL /eu;
+    AT /eu;
+    PL /eu;
+    PT /eu;
+    RO /eu;
+    SI /eu;
+    SK /eu;
+    FI /eu;
+    SE /eu;
+}
+
+server {
+    listen 80;
+    server_name  _;
+
+    add_header Cache-Control "public, max-age=7200, s-maxage=600, must-revalidate";
+    charset UTF-8;
+
+    error_page 404 /404.html;
+
+    location @custom_error_503 {
+        return 503;
+    }
+
+    location ~ /\.git {
+        return 404;
+    }
+
+    location / {
+        if ($http_cf_ipcountry) {
+            return 301 https://$host$redirected_url$request_uri;
+        }
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
```

Comment:
We'd need to use the request path for resolution of the html files here too. Currently this would serve `index.html` for all, right?